### ISSUE TITLE
[PRE REVIEW]: DiffCapAnalyzer: A Python Package for Quantitative Analysis of Total Differential Capacity Data

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#2477]
**Submitting author:** @nicolet5 (<a href="http://orcid.org/0000-0003-3411-7373">Nicole  Thompson</a>)
**Repository:** <a href="https://github.com/nicolet5/DiffCapAnalyzer" target ="_blank">https://github.com/nicolet5/DiffCapAnalyzer</a>
**Version:** v0.1.1
**Editor:** @jgostick
**Reviewers:** @yangbai90
**Managing EiC:** Arfon Smith

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**Author instructions**

Thanks for submitting your paper to JOSS @nicolet5. **Currently, there isn't an JOSS editor assigned** to your paper.

@nicolet5 if you have any suggestions for potential reviewers then please mention them here in this thread (without tagging them with an @). In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission (please start at the bottom of the list).

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
Failed to discover a `Statement of need` section in paper

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.84  T=0.20 s (117.4 files/s, 30968.3 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
Python                          14            343            812           2420
Jupyter Notebook                 4              0           1851            510
Markdown                         3             43              0            198
TeX                              1              0              0            126
YAML                             2              0              2             27
-------------------------------------------------------------------------------
SUM:                            24            386           2665           3281
-------------------------------------------------------------------------------


Statistical information for the repository '2477' was gathered on 2020/07/12.
The following historical commit information, by author, was found:

Author                     Commits    Insertions      Deletions    % of changes
Nicole Thompson                 96       1621450        1634710           98.69
Robert Masse                     8          1324             17            0.04
Sarah Alamdari                  57         11696           9928            0.66
Ted Cohen                       34          3390           2087            0.17
sarahalamdari                    2             0              2            0.00
unknown                          1          1012              0            0.03
wesleybeckner                    1         13729              0            0.42

Below are the number of rows from each author that have survived and are still
intact in the current revision:

Author                     Rows      Stability          Age       % in comments
Nicole Thompson            5223            0.3          7.3               10.24
Robert Masse                 12            0.9         23.1               16.67
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.02477/joss.02477/10.21105.joss.02477.pdf)

**Comment by nicolet5** [comment]
A potential reviewer: Logan Ward (WardLT)

**Comment by Kevin-Mattheus-Moerman** [comment]
@jgostick @katyhuff @richardjgowers @kyleniemeyer would one of you be able to handle this submission?

**Comment by katyhuff** [comment]
I'm pretty over-committed over the next month. If someone else could take it, I would appreciate that.

**Comment by kthyng** [comment]
Hi @jgostick could you edit this one?

**Comment by kthyng** [comment]
@whedon invite @jgostick as editor

**Comment by whedon** [comment]
@jgostick has been invited to edit this submission.

**Comment by nicolet5** [comment]
@labarba Thank you for pointing that out! I wasn't previously aware that was a needed section. I have updated the paper to include the _Statement of need_ section.

**Comment by jgostick** [comment]
@whedon assign @jgostick as editor

**Comment by whedon** [comment]
OK, the editor is @jgostick

**Comment by jgostick** [comment]
@WardLT, you have been suggested as a potential reviewer for this package.  I don't have your email address, otherwise I'd contact you directly.  If you're interested and available, please let me know by responding this thread.  If I don't hear from you in 7 days I'll look for others.  Thanks!

**Comment by jgostick** [comment]
Hello @yangbai90, you have listed yourself as volunteer JOSS reviewer, and have listed 'battery modeling' as an area of expertise.  I was hoping you'd be interested in reviewing this contribution, since it's about batteries...though more from a systems perspective and less about FEM based performance.  Please let me know here in this thread or contact me by the email address associated with my Github account.

**Comment by yangbai90** [comment]
@jgostick Hello Prof. Gostick, I'm very interested in your paper, I will do the reviewing for it.

**Comment by WardLT** [comment]
Sure, I'll gladly review. How do I get started?

**Comment by jgostick** [comment]
@whedon assign @WardLT as reviewer

**Comment by whedon** [comment]
OK, @WardLT is now a reviewer

**Comment by jgostick** [comment]
@whedon assign @yangbai90 as reviewer

**Comment by whedon** [comment]
OK, @yangbai90 is now a reviewer

**Comment by jgostick** [comment]
@whedon start review

**Comment by whedon** [comment]
OK, I've started the review over in https://github.com/openjournals/joss-reviews/issues/2624.

**Comment by jgostick** [comment]
@arfon something odd happened on this issue when I assigned reviewers.  I assigned @WardLT, then assigned @yangbai90, and whedon did something unexpected (I think) as you can see by his/her comments above.  Is this buggy or did I screw up?

**Comment by arfon** [comment]
@jgostick `assign` clobbers all existing assignments, `add` adds a new reviewer so your second comment here: https://github.com/openjournals/joss-reviews/issues/2477#issuecomment-684181713 removed the first reviewer. 

You can fix this over in #2624 by:

1. Running `@whedon add @WardLT as reviewer`
2. Editing the issue at the top of the thread to add a second checklist for @WardLT 

Are you OK to do this? If not, I can help.
